cornell-dti/cu-apts
# Initial Implementation of FAQ Page

## Summary
- Closes issue #16.
- This pull request is for the basic implementation of the FAQ page that is linked on the Home page.

### Before/After
- The "FAQ" button on the Home page now links to this FAQ page. 
- The CU Housing logo is in the top lefthand corner of the page, and the top righthand corner has a link to the "Reviews" page that will be implemented in the future. 
- The "Need Help?" section is in the middle of the page with a placeholder search bar until designs are finalized, and the collapsible FAQ section appears below this. 
- The FAQ section is still populated using hardcoded data. The new backend routes and frontend/backend connection will need to be merged in.

### Test Plan
- Run `yarn install` in `cu-apts` to make sure all packages are installed.
- Run `yarn start` in `cu-apts` to view the React project in a local browser. It will open to the Home page, so click on the "FAQ" button on the top righthand corner to view the FAQ page.
